src/builtins/danger_zone.rs

```diff
@@ -125,3 +125,23 @@ impl RuleSet for ForkAndExec {
         "ForkAndExec"
     }
 }
+
+/// [`Pipes`] is in the danger zone because it can be used create a pipe for IPC.
+///
+/// # Security Considerations
+/// The piped process will still be under seccomp's restrictions (see
```

Comment:
Sorry again for not getting to this faster. This still isn't correct even without sending fds via unix domain sockets.

You can, for example, create the pipe, fork the process, and then the parent and the child both have the pipe but even if you apply seccomp to the parent, it doesn't apply to the already-forked child (unless you use `apply_to_all_threads` of course)

Reply:
Sorry about that, I'll adjust the docs accordingly.